README.md

```diff
@@ -285,6 +285,12 @@ environment variable. We first document the most relevant and most commonly used
   `TERM` environment variable is excluded by default to [speed up the test
   harness](https://github.com/rust-lang/miri/issues/1702). This has no effect unless
   `-Zmiri-disable-isolation` is also set.
+* `-Zmiri-extern-so-file=<path to a shared object file>` is an experimental flag for providing support
+  for FFI calls.
+  **WARNING**: If an invalid/incorrect SO file is specified, this can cause undefined behaviour in Miri itself!
```

Comment:
```suggestion
  **WARNING**: If an invalid/incorrect `.so` file is specified, this can cause undefined behaviour in Miri itself!
  And of course, Miri cannot do any checks on the actions taken by the external code.
```